.github/workflows/test-docs.yml

```diff
@@ -1,20 +1,19 @@
-# ATTENTION: Although this workflow doesn't do much, it is still important.
-# It is complementary to the workflow in the file test-code.yml.
-# It enables to exclude files for the workflow and still mark the Test job as required without having pending PRs.
-# GitHub recommends this solution here:
-# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
-
 name: ZITADEL PR
 
 on:
   pull_request:
     paths:
-      - 'docs/**'
-      - 'guides/**'
-      - '**.md'
+      - "docs/**"
+      - "**.md"
 
 jobs:
   Test:
     runs-on: ubuntu-20.04
     steps:
-      - run: 'echo "No tests for docs are implemented, yet"'
+      - name: Source checkout
+        uses: actions/checkout@v2
+      - name: Check formatting
+        uses: creyD/prettier_action@v4.2
+        with:
+          prettier_version: "2.7.1"
+          prettier_options: --check docs/docs --check **/*.md
```

Comment:
should this not also check mdx?

Reply:
Good catch

Reply:
I think it has no effect, as we don't have .mdx outside the docs folder. But it's more complete if we add .mdx 👍